core/src/main/scala/sbt/librarymanagement/ResolverExtra.scala

```diff
@@ -162,27 +162,27 @@ abstract class ResolverFunctions {
   def jcenterRepo = JCenterRepository
 
   /** Add the local and Maven Central repositories to the user repositories.  */
-  def withDefaultResolvers(userResolvers: Seq[Resolver]): Seq[Resolver] =
-    withDefaultResolvers(userResolvers, mavenCentral = true)
+  def defaultResolvers(userResolvers: Vector[Resolver]): Vector[Resolver] =
```

Comment:
Note that this gets mixed into the companion object Resolver. So I think @Duhemm is saying it makes it sound like it's a build pattern name but it's not, as you can see from the signature returning Vector of Resolver.

See also https://github.com/sbt/librarymanagement/pull/124#issuecomment-314407061 

> I don't think defaultResolvers is good since it's returning more than the default.
combineDefaultResolvers? (since we can't say for sure if it's prepend or append)